osbs/api.py

```diff
@@ -740,7 +740,7 @@ def get_token(self):
             if self.os.token:
                 return self.os.token
 
-            raise OsbsValidationException("no token stored")
+            raise OsbsValidationException("no token stored for %s" % self.os_conf.conf_section)
```

Comment:
test is catching exception, not message, I don't think it is worth to checking that